mpas_analysis/test/test_climatology.py

```diff
@@ -296,6 +296,9 @@ def test_cache_climatologies(self):
                                                             printProgress=True)
             self.assertArrayApproxEqual(dsClimatology.mld.values,
                                         refClimatology.mld.values)
+
+            self.assertEqual(dsClimatology.attrs['totalMonths'], 3)
+            self.assertApproxEqual(dsClimatology.attrs['totalDays'], 89.958333)
```

Comment:
I added a comment.  Beta0 data sets like the one I used here had `xtime_start` as one timestep after the beginning of the month.  This appears to have been fixed in Beta1.